eng/Version.Details.xml

```diff
@@ -30,6 +30,10 @@
       <Uri>https://github.com/aspnet/xdt</Uri>
       <Sha>d4d088b6a9c793525b1a27a119cb66ba4587bb39</Sha>
     </Dependency>
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.24068.1">
```

Comment:
This doesn't match our built against version. 2.0.0-beta4.23307.1

Is that intentional? 
We matched our version to the one in the .NET SDK recently.

Reply:
That's OK - versions in this file are used for source-build only.